user-access-handlers/build.gradle

```diff
@@ -1,17 +1,18 @@
 dependencies {
 
-    implementation project(':user-access-commons')
-    implementation project(':user-access-public-model')
+    compileOnly project(':user-access-commons')
```

Comment:
implementation means not use of the "AWS::Serverless::LayerVersion" resource. We use compileOnly to exclude the libraries from the package